app/models/DeclarationFormats.scala

```diff
@@ -71,4 +72,6 @@ object DeclarationFormats {
   implicit val loadingLocationFormats = Json.format[LoadingLocation]
 
   implicit val warehouseFormats = Json.format[Warehouse]
-}
+
+  implicit val governmentAgencyGoodsItemFormats = Jsonx.formatCaseClass[GovernmentAgencyGoodsItem]
```

Comment:
Why you need this? I mean whole `Jsonx`?

Reply:
this is the case class which has 21+ fields and  I sued the library to handle implicit Json formats

Reply:
IMHO we should raise a problem on data model once again. We continue to be highly dependent on wco-dec repository, while we should be loosely coupled to anything from outside. We could be more independent with our logic as well.

Reply:
I think that we should get rid of those classes, but I understand that for know probably only this `Jsonx` will solve a problem. Just we need to start preparing model with representation of our declaration, not `wco-dec`

Reply:
we will raise a tech debt story once we have a view on the most of declaration types until then ItemCachingService will serve a place for all complicated transformation.